graylog2-web-interface/src/theme/GraylogThemeProvider.jsx

```diff
@@ -37,13 +37,13 @@ const createTheme = (mode, themeColors, changeMode): ThemeInterface => {
   };
 };
 
-const GraylogThemeProvider = ({ children }: Props) => {
-  const colorScheme = usePrefersColorScheme();
-  const [mode, setCurrentThemeMode] = useCurrentThemeMode(colorScheme);
+const GraylogThemeProvider = ({ children, defaultMode }: Props) => {
+  const [mode, setCurrentThemeMode] = useCurrentThemeMode();
+  const themeMode = defaultMode ?? mode;
```

Comment:
Maybe a conditional here, if `defaultMode` is set, no reason to do the `useCurrentThemeMode`?

Reply:
That would break the first rule for hooks, not calling them from the top-level/unconditionally imo. Might work right now, but could introduce subtle bugs in the future if `GraylogThemeProvider` gets more state.